cosmwasm/packages/wasmi-runtime/src/macros.rs

```diff
@@ -51,3 +51,27 @@ macro_rules! coalesce {
         wrapper()
     }};
 }
+
+#[macro_export]
+macro_rules! validate_const_ptr {
+    ($ptr:expr, $ptr_len:expr) => {{
+        if let Err(_e) = validate_const_ptr($ptr, $ptr_len) {
```

Comment:
`validate_const_ptr` should be fully qualified as `$crate::utils::validate_const_ptr`